src/js/constants.js

```diff
@@ -64,3 +70,8 @@ export const ReadyStates = {
   HAVE_FUTURE_DATA: 3,
   HAVE_ENOUGH_DATA: 4,
 };
+
+export const AvailabilityStates = {
```

Comment:
Don't hate me... I think I'm convinced we should just separate these into two attributes, -unsupported and -unavailable. Seeing the varying use cases on all sides of the equation has helped.

Reply:
All good. Let's discuss tradeoffs in our 1:1.

Reply:
After a chat we decided to stick with a single attr for now, for the general simplicity of handling both situations at once.